bruno/bruno.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <title>Bruno Cerqueira - Curriculum</title>
+</head>
+<body>
+    <h1>Bruno Almeida Cerqueira</h1>
+    
+    <img src="photo.JPG" alt="photo curriculum">
+    
+    <h2>Personal Data</h2>  
+    <p>
+        23 years, single </br>
```

Comment:
The usage of `<br />` is incorrect in your document. Please use `<br>` or `<br />` instead of `</br>`.